### PR TITLE
fix #59032 file-upload.com

### LINKED ISSUE #59032: file-upload.com
### Issue URL (Annoyance)
 [https://www.file-upload.com/wn3mjjzaqeqt](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.file-upload.com%2Fwn3mjjzaqeqt)
### Comment
Username: @blackrider020
> This iframe is causing annoying pop-ups to open in a new tab. After I delete this iframe in dev tool, there's no pop-ups. 
> 
> Thank you!

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese, <br/>EasyList China, <br/>EasyList Italy, <br/>Liste AR

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#59032

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -3,6 +3,9 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59035
 comicastle.org#?#.card > .card-body.text-center:has(> .adsbygoogle)
+! https://github.com/AdguardTeam/AdguardFilters/issues/59032
+file-upload.com#?#.container > div.page-wrap > div.text-center[style^="margin-bottom: 20px;"]:contains(ads)
+file-upload.com#?#.container > div.page-wrap > center:contains(ads)
```
Comment (Alex-302) [inline EnglishFilter/sections/css_extended.txt:8]: I don't see these elements.

File: EnglishFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -66,7 +66,7 @@ xxxstreams.me,watchfreexxx.net,playpornfree.org,pandamovie.co,losporn.net,nameth
 xxxparodyhd.net,uptobox.com,uptostream.com,pandamovies.me,pornkino.cc,watchpornfree.info,xxxmoviestream.xyz,mangoporn.co,pandavideo.pw,playpornfree.xyz,xopenload.me,streamporn.pw#%#//scriptlet("abort-on-property-read", "jsUnda")
 !
 ! For scripts, which uses base64
-force-download.es,bro.adca.st,jpost.com,convert-me.com,keezmovies.com#%#window.atob = function() { };
+file-upload.com,force-download.es,bro.adca.st,jpost.com,convert-me.com,keezmovies.com#%#window.atob = function() { };
```
Comment (Alex-302) [inline EnglishFilter/sections/general_extensions.txt:69]: better add to existing rule `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")`
And chech for unnecessary JS for this site(just list the rules, don't make changes in pull-request).

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/general_extensions.txt:69]: > better add to existing rule `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")`

I don't see this rule

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/css_extended.txt:8]: you can see in on last page before download
<details>
<summary>Screenshot</summary>

</details>

**Pull request closed without merging**